chains/cosmoshub/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=../schema.json
+bech32Prefix: cosmos
+blockExplorers:
+  - apiUrl: https://www.mintscan.io/cosmos
```

Comment:
Is this correct here? I see they have these docs for the API https://docs.cosmostation.io/apis

Reply:
It doesn't actually matter b.c. we don't use apiUrls for explorer families other than Etherscan atm. But you're correct that there's now a real api so I've updated all the minscan block explorers :)